examples/test_suite.sh

```diff
@@ -195,6 +195,10 @@ fi
 if [ "x${BUILDDIR}" != "x" ]; then
   LIBS_FROM_BUILDDIR=$(find ${BUILDDIR} -iname "*.so" -exec dirname {} \; | tr '\r\n' ':')
   export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${LIBS_FROM_BUILDDIR}
+  # In that case, also use headers from matching source dirs.
+  SOURCEDIR=$(readlink -f ../source/)
+  HEADERS_FROM_SOURCEDIR=$(find ${SOURCEDIR} -iname "*.h" -exec dirname {} \; | tr '\r\n' ':')
```

Comment:
You may want to pipe the output of find through `uniq` to remove duplicate entries.

Reply:
Great catch! 
I added a second commit which does this for both the `LD_LIBRARY_PATH` (there it is "mostly harmless" due to the low number of libraries) and the newly added code.

Reply:
Thank you, @olifre , for the fix and sorry for a delay.